libnmstate/netapplier.py

```diff
@@ -28,7 +28,6 @@
 from libnmstate.nm.nmclient import nmclient_context
 from libnmstate import state
 from libnmstate import validator
-from libnmstate.deprecation import _warn_keyword_as_positional
```

Comment:
Please also remove the code for `_warn_keyword_as_positional`, it seems to be kept.